internal/buildscripts/packaging/msi/splunk-support-bundle.ps1

```diff
@@ -0,0 +1,263 @@
+# Copyright Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#######################################
+# Globals
+#######################################
+$CONFDIR="${env:PROGRAMFILES}\Splunk\OpenTelemetry Collector" # Default configuration directory
```

Comment:
Sorry, the default config dir for the collector should be:

```suggestion
$CONFDIR="${env:PROGRAMDATA}\Splunk\OpenTelemetry Collector" # Default configuration directory
```

We should also include the td-agent config dir in the bundle, which defaults to `${env:SYSTEMDRIVE}\opt\td-agent\etc\td-agent`.